### PR TITLE
[ShapeOpt] New example and readmes

### PR DESCRIPTION
**Pull request by ge73xip**
- example for multi cosntrained hook model
- example for surface wrapping around stanford bunny

### CODE REVIEW

File: shape_optimization/10_Multi_Constraint_Optimization_3D_Hook/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+**Author**: Armin Geiser  
+
+**Kratos version**: 9.0
+
+## Optimization Problem
+- Objective
+    - Minimize mass
+
+- Constraints
+    - constraint 1  &nbsp; : &nbsp; strain energy of main &le; initial value for load case 1
```
Comment (armingeiser) [inline shape_optimization/10_Multi_Constraint_Optimization_3D_Hook/README.md:13]: Please don't use these spacings (`&nbsp`) makes the raw text hard to read and is usually not needed.

same holds for items below.

File: shape_optimization/10_Multi_Constraint_Optimization_3D_Hook/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    - constraint 2  &nbsp; : &nbsp; strain energy of tip &le; initial value for load case 2
+    - constraint 3  &nbsp; : &nbsp; mesh based packaging &le; specified value (use of bounding mesh)  
+
+        <img src="images/3D_Hook_Constraints.svg" height="500">
```
Comment (armingeiser) [inline shape_optimization/10_Multi_Constraint_Optimization_3D_Hook/README.md:17]: Here the right part of the image is enough.

File: shape_optimization/10_Multi_Constraint_Optimization_3D_Hook/README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+- Filter Radius &nbsp; &nbsp; : &nbsp; 25.0
+
+- Mesh motion &nbsp; : &nbsp; YES 
+
+## Optimization algorithm
+- type &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; : &nbsp; gradient_projection
+- number of steps &nbsp;: &nbsp; 20 &nbsp;
+- step size &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; : &nbsp; 3.0
+
+## Results
```
Comment (armingeiser) [inline shape_optimization/10_Multi_Constraint_Optimization_3D_Hook/README.md:28]: I would add two subsections:
- Shape evolution
    - here add the left part of the above image (ideally as a gif) The text overlay is not necessary, we can also describe the content below the image. 
- Convergence Plots

File: shape_optimization/10_Multi_Constraint_Optimization_3D_Hook/README.md (hunk 1 above)
Comment (armingeiser) [reply shape_optimization/10_Multi_Constraint_Optimization_3D_Hook/README.md:13]:
```suggestion
    1. strain energy of main load (LC1) &le; initial value
```

**Comment by armingeiser** [comment]
- links to folder
- add in main readme
- improve font in graph

File: shape_optimization/11_Shape_Update_Optimization_Stanford_Bunny/README.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
+# Shape Optimization
+Shape update of a wrapping surface to generate a smooth representation of a complex object, which in this example is a Stanford bunny.
```
Comment (armingeiser) [inline shape_optimization/11_Shape_Update_Optimization_Stanford_Bunny/README.md:2]: For this example please describe only the small sphere example, as this is the one with better results and it is also the one that is in the top level of the folder.

please crop the gif (https://ezgif.com/crop) and figures such that only that example is shown. 

At the end of the page we can make a small reference to the shrink folder by only showing the gif of the shrinking and stating tat with slight modifications of the optimization setup a shrinking can be achieved.

File: shape_optimization/11_Shape_Update_Optimization_Stanford_Bunny/README.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+### Objective
+- Large sphere: shape_update minimization 
+- Small sphere: shape_update maximization
+
+### Constraints
+- No penetration of packaging (bounding) mesh (*standford bunny*)
+
+  <p align="center">
+    <img src="images/bunny_opt_setup.png" height="500">
```
Comment (armingeiser) [inline shape_optimization/11_Shape_Update_Optimization_Stanford_Bunny/README.md:19]: please make the annotated text smaller, it should be approx the font size of the normal markdown text

File: shape_optimization/11_Shape_Update_Optimization_Stanford_Bunny/README.md (hunk 4 above)
Comment (ge73xip) [reply shape_optimization/11_Shape_Update_Optimization_Stanford_Bunny/README.md:2]: shrink is with Large sphere & growth would be the Small sphere ? right ?

Comment (armingeiser) [reply shape_optimization/11_Shape_Update_Optimization_Stanford_Bunny/README.md:2]: yes

**Comment by armingeiser** [comment]
@ge73xip Thanks, looks very good now!
For the next update please create a new PR.

**Pull request merged**